app/src/main/java/org/mozilla/fenix/library/history/HistoryController.kt

```diff
@@ -25,10 +25,14 @@ interface HistoryController {
     fun handleDeleteSome(items: Set<History>)
     fun handleRequestSync()
     fun handleEnterRecentlyClosed()
+    /**
+     * Navigates to [org.mozilla.fenix.library.syncedhistory.SyncedHistoryFragment]
```

Comment:
```suggestion
     * Navigates to [SyncedHistoryFragment].
```

Reply:
It won't work as a link that way, will it?

Reply:
You can import the full path.